specs/latest/1.0/index.html

```diff
@@ -4394,6 +4394,32 @@ <h3>Uniform and attribute name collisions</h3>
     drivers don't accept uniforms and vertex attributes with the same name.
 </div>
 
+<h3>Wide point primitive clipping</h3>
+<p>
+    <code>POINTS</code> primitives may or may not be discarded if the vertex lies outside the clip
+    volume, but within the near and far clip planes.
+</p>
+<div class="note rationale">
+    <p>
+        Clipping of wide points works differently in GLES and GL, and this difference in behavior is
+        prohibitive to work around in implementations.
+    </p>
+    <p>
+        OpenGL ES 2.0.25 p46:
```

Comment:
Possible to add a hyperlink to https://www.khronos.org/registry/OpenGL/specs/es/2.0/es_full_spec_2.0.pdf ?